plugins/modules/remote_management/hpilo/hpilo_info.py

```diff
@@ -122,6 +122,226 @@
     type: str
     sample: ON
     version_added: 3.5.0
+
+server_name:
+    description: Server name
```

Comment:
The description should always have complete sentences.
```suggestion
    description: The server name.
```